.github/BOTMETA.yml

```diff
@@ -450,7 +450,7 @@ files:
     maintainers: dmtrs
   $modules/consul:
     ignore: colin-nolan Hakon
-    maintainers: $team_consul
+    maintainers: $team_consul Ilgmi
```

Comment:
Maybe move yourself into $team_consul, I don't think it's referenced somewhere else anyways and so all users would be in one place.